### PR TITLE
Release 1.85.2

### PR DESCRIPTION
**Pull request by singholt**
### Testing

New tests cover the changes: 

### Description for the changelog

**Does this PR include breaking model changes? If so, Have you added transformation functions?**
  

### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
 # Changelog
+# 1.85.2
+Bugfix - Disable Digest Resolution for Manifest V2 Schema 1 [#4249](https://github.com/aws/amazon-ecs-agent/pull/4249)
```
Comment (tinnywang) [inline CHANGELOG.md:3]: The rich diff is showing a paragraph of text because each line is missing the `*` that makes it a list item.

**Review by harishxr — APPROVED** [review]
Let's get the changelog fixed in a follow up PR.

**Review by mye956 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (amogh09) [inline CHANGELOG.md:3]: #4249 is not really a bugfix. It's a mitigation for a bug in ECR.

Comment (amogh09) [reply CHANGELOG.md:3]: Not sure what else we can label it though.

Comment (singholt) [reply CHANGELOG.md:3]: agreed. 

"Mitigation" ? It would be a first in the changelog.

**Pull request merged**